src/pipes/db-lookup.pipe.ts

```diff
@@ -4,42 +4,31 @@ import {
   ValidationArguments,
 } from "class-validator";
 import { FindOneOptions } from "typeorm";
-import { DbLookupValidator } from '../validators';
+import { DbLookupValidator } from "../validators/db-lookup.validator";
+
+export interface DbLookupConfig {
+  type: any;
+  findOption?: ((validationArguments: ValidationArguments) => FindOneOptions) | "primary";
+  validateNumeric?: boolean;
+  ignoreEmpty?: boolean;
+}
```

Comment:
Unfortunately, moving these all into an object is _not_ backwards compatible. I think the least intrusive way would be to just use the last option for `DbLookup`-related options:
```suggestion
export interface DbLookupConfig {
  validateNumeric?: boolean;
  ignoreEmpty?: boolean;
}
```

Reply:
```markdown
Pls see [https://github.com/US-EPA-CAMD/easey-common/pull/175](url)